<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProductsElaboratedAdd</class>
 <widget class="QDialog" name="frmProductsElaboratedAdd">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Managing elaborated products</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/caloriestracker.qrc">
    <normaloff>:/caloriestracker/books.png</normaloff>:/caloriestracker/books.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_16">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 128, 0);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Name of the product</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Final amount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="myQDoubleSpinBox" name="spnFinalAmount">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string> g</string>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="myQTableWidget" name="tblProductsIn"/>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="bb">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">myqdoublespinbox</header>
  </customwidget>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/caloriestracker.qrc"/>
 </resources>
 <connections/>
</ui>
